torch/lib/TH/generic/THTensorMath.c

```diff
@@ -1927,6 +1927,18 @@ void THTensor_(zeros)(THTensor *r_, THLongStorage *size)
   THTensor_(zero)(r_);
 }
 
+void THTensor_(zeros_like)(THTensor *r_, THTensor *input)
+{
+  THTensor_(resize)(r_, THTensor_(newSizeOf)(input), NULL);
```

Comment:
I think this will leak `THLongStorage*` returned by `newSizeOf`.

Instead:

`THTensor_(resizeAs)(r_, input)`